lib/cea/cea_parser.js

```diff
@@ -0,0 +1,38 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.cea.CeaParser');
+
+/**
+ * Interface for parsing inband closed caption data from MP4 streams.
+ * @interface
+ */
+shaka.cea.CeaParser = class {
+  /**
+   * Initializes the parser with stream segment data.
+   * @param {BufferSource} data Stream segment data to parse.
+   */
+  init(data) {}
+
+  /**
+   * Parses the stream and extracts 708 closed captions information.
+   * @param {BufferSource} data segment data
+   * @param {function(!BufferSource, number)} on708Data called when the parser
```

Comment:
It is unclear what these two callback parameters are.  I could make a guess about the BufferSource param, but best if it's documented.  And I have no clue what the number might be.